src/components/EditCoursePage/CollapsibleCourseRuns.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import CollapsibleCourseRun from './CollapsibleCourseRun';
+
+const CollapsibleCourseRuns = ({
+  courseRuns,
+  fields,
+  ...others
```

Comment:
I think we should have a quick team discussion to standardize how we want to name props that are just meant to be passed down. I've seen `others`, `extraInput`, and `passedProps` all in this PR

Reply:
`extraInput` and `passedProps` are both shapes themselves with specific targets in mind. Like `extraInput` is aimed at the underlying `input` element.

So those I could see being looser about naming - they have their own intent they are trying to convey.

But in this case, we are looking for something that means "the rest of the props". Would `otherProps` be nicer? `props` by itself feels too ambiguous.

Reply:
I still feel like at the end of the day, both extraInput and passedProps are props we are handing down the chain, but I do see your point.

Agreed that props by itself is too ambiguous. `otherProps` is nicer, but I don't know. Does FedX have any thoughts on this?

https://reactjs.org/docs/higher-order-components.html#convention-pass-unrelated-props-through-to-the-wrapped-component Calls is `passThroughProps`, but I'm not in love with that either

Reply:
`passThroughProps` is probably just an example. But it is in the official docs on how to do this. And it is explicit. I'm fine with it.

Reply:
Blargh. I will accept any executive decision made.

Reply:
I just used `passThroughProps` and fixed a few other instances of other names.

Reply:
I think the important thing is consistency - easier to find all instances and rename them in future, if they all use `passThroughProps` for now.